documenting.rst

```diff
@@ -1578,14 +1578,242 @@ Sphinx_, blurb_ and python-docs-theme_ packages from PyPI)::
+|              |                    |                                          |
++--------------+--------------------+------------------------------------------+
+| Spanish      | Raul Cumplido      | `github <https://github.com/raulcd/      |
+| (es)         |                    | python-docs.es/>`__                      |
+|              |                    |                                          |
+|              |                    | `old repo <https://github.com/PyAr/      |
+|              |                    | tutorial/>`__                            |
++--------------+--------------------+------------------------------------------+
+| Traditional  | 廖偉涵 Adrian Liaw | `github <https://github.com/python/      |
+| Chinese      |                    | python-docs-zh-tw/>`__                   |
+| (zh-tw)      |                    |                                          |
+|              |                    | `transifex <https://www.transifex.com/   |
+|              |                    | python-tw-doc/python-36-tw>`__           |
+|              |                    |                                          |
+|              |                    | `doc <https://docs.python.org/zh-tw/>`__ |
++--------------+--------------------+------------------------------------------+
+| Turkish (tr) |                    | `github <https://github.com/alaeddingurel|
+|              |                    | /python-docs-tr>`__                      |
+|              |                    |                                          |
++--------------+--------------------+------------------------------------------+
+
+
+Starting a new translation
+--------------------------
+
+First subscribe to the `doc-sig
+<https://mail.python.org/mailman/listinfo/doc-sig>`_ mailing list,
+introduce yourself and the translation you're starting.
+
+Then you can bootstrap your new translation by using our `cookiecutter
+<https://github.com/JulienPalard/python-docs-cookiecutter>`__.
+
+The important steps looks like this:
+
+- Create the github repo (anywhere), with the right hierarchy (using the
+  cookiecutter).
+- Gather people to help you translating, you can't do it alone.
+- You can use any tool to translate, as long as you can synchronize with git,
+  some are using Transifex, some are using only github, you can choose another
+  way, it's up to you.
+- Ensure we updated this page to reflect your work and progress, either via a
+  PR, or by asking on the doc-sig mailing list.
+- When ``tutorial/``, ``library/stdtypes`` and ``library/functions``
+  are complete, ask on doc-sig for your language to be added in the
+  language picker on docs.python.org.
+
+
+PEP 545: TL;DR
+--------------
+
+The :PEP:`545` may be long to read, so here are the interesting points:
+
+- Each translation is assigned an appropriate lowercased language tag,
+  with an optional region subtag (if needed), glued with a dash, like
+  ``pt-br`` or ``fr``.
+
+- Translations files are hosted on
```

Comment:
Well tbh only the ones advanced enough, not all of them 🤔 how can we reflect that ?